moveit_core/constraint_samplers/test/pr2_arm_ik.cpp

```diff
@@ -49,6 +49,8 @@
 using namespace angles;
 using namespace pr2_arm_kinematics;
 
+rclcpp::Logger LOGGER_PR2_ARM_IK = rclcpp::get_logger("pr2_arm_kinematics_plugin");
```

Comment:
Place inside namespace in .hpp

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/d9872d1ad04537b9aa2617c438095fde9c524bf3